src/getPage.tsx

```diff
@@ -143,5 +149,6 @@ export default async function getPage({
     // @NOTE Temporary hack to keep tests green
     page: useDocument ? serverPageElement : clientPageElement,
```

Comment:
Is this "hack" still needed?

Reply:
I'm currently working on a 3rd branch to integrate document render into the new rendering flow.